### DISCUSSION TITLE
Windows11 22H2 taskbar

### DISCUSSION THREAD

**Discussion by bao-cn** [ramensoftware/windhawk-mods#60] [Mod ideas]
Windows11 22H2 has modified the relevant content of the task bar. When writing to switch the color mode by time, there will be no response. Click the start button of the task bar to change the color. After the color change, there will be no change in the font color. Using the win32 function to refresh the window area also has no response. What should I do?

**Comment by m417z** [comment]
Moving the discussion to windhawk-mods, since the Windows 11 taskbar isn't directly related to Windhawk itself.

I don't quite understand the context of your message. Are you suggesting that a Windhawk mod could solve the problem you're describing?
